content/doc/book/installing/index.adoc

```diff
@@ -140,6 +140,74 @@ about this in link:../blueocean/getting-started[Getting started with Blue Ocean]
 
 include::doc/book/installing/_docker.adoc[]
 
+=== Kubernetes
+
+Kubernetes (K8s) is an open-source system for automating deployment, scaling, 
+and management of containerized applications. 
+
+A Kubernetes cluster adds a new automation layer to Jenkins. 
+Kubernetes makes sure that resources are used effectively and that your servers 
+and underlying infrastructure are not overloaded.
+Kubernetes’ ability to orchestrate container deployment ensures that Jenkins always 
+has the right amount of resources available.
+This section will describe different options to install/run Jenkins on Kubernetes. 
+
+==== Prerequisites
```

Comment:
It would be nice to highlight that it is a QuickStart here. The documented approach with Minikube should not be used for production instances, hence it makes sense to refer the full K8s documentation